apps/server/src/shared/contract.ts

```diff
@@ -1,3 +1,4 @@
+import type { TEST_EXAMPLE_A_PROVIDER_STRING } from '../test-adapter'
```

Comment:
I don't see this type used in this file.

Reply:
Good catches